synthtool/gcp/common.py

```diff
@@ -49,10 +49,10 @@ def __init__(self, template_path: Optional[Path] = None):
         self._templates = templates.Templates(self._template_root)
         self.excludes = []  # type: List[str]
 
-    def _generic_library(self, directory: str, **kwargs) -> Path:
+    def _generic_library(self, directory: str, relative_dir: str, **kwargs) -> Path:
```

Comment:
should `relative_dir` have a default value, it looks like it will be undefined in some cases?